### PR TITLE
fix(activity): show `runEnd` time rather than `commitTime` in the activity log

### PR DESCRIPTION
**Pull request by ramfox**
Also, adds logic to see if a timestamp is actually the "zero" value before displaying it. If it is the "zero" value (or is somehow invalid), instead return "--"

### CODE REVIEW

File: src/utils/runEndTime.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
 export function runEndTime(runStart: string, runDuration: number): Date {
   const runStartTime = new Date(runStart)
-  return new Date(runStartTime.getTime() + (runDuration / 1000000))
+  if (!runDuration) runDuration = 0
```
Comment (chriswhong) [inline src/utils/runEndTime.ts]: Add curly brackets even for one line if statements

File: src/utils/runEndTime.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
+  const d =  new Date(runStartTime.getTime() + (runDuration / 1000000))
+  return d
```
Comment (chriswhong) [inline src/utils/runEndTime.ts]: Maybe just `return new Date(runStartTime.getTime() + (runDuration / 1000000))`

**Review by chriswhong — APPROVED** [review]
2 nits, then LGTM

File: src/utils/runEndTime.ts (hunk 2 above)
Comment (ramfox) [reply src/utils/runEndTime.ts]: ahh yes, ty. Hold over from when I was testing & needed log the output

**Pull request merged**